src/schematics/src/update/update-0.13.0.ts

```diff
@@ -76,6 +76,16 @@ Start changes for Clarity 0.13.0
             );
           }
 
+          // Alert to using clrType in hide/show panel in datagrid (https://github.com/vmware/clarity/pull/2546)
+          if (updated.search(/clrType/g) > -1) {
```

Comment:
This seems too generic, let's make it match something like `clrType="(.*?)"` (not correct regex)

Reply:
Sounds reasonable. Will update the regex.